docs/source/reference/deprecation.rst

```diff
@@ -539,3 +539,56 @@ Schedule
   raise deprecation warnings.
 - In Numba 0.60: ``NUMBA_CAPTURED_ERRORS=new_style`` became the default.
 - In Numba 0.61: support for ``NUMBA_CAPTURED_ERRORS=old_style`` was removed.
+
+.. _cuda-target-deprecation-notice:
```

Comment:
```suggestion
.. _cuda-builtin-target-deprecation-notice:
```
This ends up being rendered as part of the URL (as the anchor?) which means it's sort-of user facing, suggest the minor change so that anyone copy pasting the URL into e.g. a blog post will see is as the builtin-target being deprecated. I think there's a reference from another page to this anchor which will also need the same update applying.

Reply:
Sure, I can change this and update the reference.